cmd/ci-operator/main.go

```diff
@@ -534,6 +537,31 @@ func (o *options) initializeNamespace() error {
 		break
 	}
 
+	if o.givePrAuthorAccessToNamespace {
+		// Generate rolebinding for all the PR Authors.
+		for _, pull := range o.jobSpec.Refs.Pulls {
+			rbacClient, err := rbacclientset.NewForConfig(o.clusterConfig)
+			if err != nil {
+				return fmt.Errorf("could not get RBAC client for cluster config: %v", err)
+			}
+
+			log.Printf("Creating rolebinding for user %s in namespace %s", pull.Author, o.namespace)
+			if _, err := rbacClient.RoleBindings(o.namespace).Create(&rbacapi.RoleBinding{
+				ObjectMeta: meta.ObjectMeta{
+					Name:      "temp-namespace-admin",
```

Comment:
There’s nothing temporary about this, give it a clearer name like “ci-op-author-access”